clients/extension/styles/lighthouse.css

```diff
@@ -30,181 +33,149 @@
 html, body {
   padding: 0;
   margin: 0;
-  width: 390px;
-  height: 200px;
-  background-color: var(--lh-bg-color);
+  width: 270px;
+  background-color: var(--color-bg);
   color: #212121;
+  font-family: var(--report-font-family);
+  font-size: var(--font-size);
+  line-height: 18px;
+  user-select: none;
 }
 
-.header {
-  padding: 26px;
-  display: flex;
-  flex: 1;
+body {
+  padding: 16px;
 }
 
 .header__icon {
-  margin-right: 16px;
+  margin: 2px auto 0;
 }
 
-.header-titles {
-  display: flex;
-  flex-direction: column;
-  align-self: center;
-  flex: 1;
-  overflow: hidden;
-  text-overflow: ellipsis;
+.errormsg {
+  color: var(--color-blue);
+  font-weight: 500;
+  text-align: center;
+}
+.errormsg:empty {
+  display: none;
 }
 
-.header-titles__main {
-  margin: 0;
-  font-family: 'Roboto', Arial, sans-serif;
-  color: currentColor;
-  font-weight: 300;
-  margin-bottom: 4px;
+.button {
+  background-color: transparent;
+  border: none;
+}
+.button:focus {
+  outline: none;
 }
 
-.header-titles__url {
-  margin: 0;
-  font-size: 14px;
-  opacity: 0.74;
-  color: currentColor;
+.button--generate {
+  cursor: pointer;
   font-family: 'Roboto', Arial, sans-serif;
```

Comment:
i think this is duplicated by the html,body font already here

Reply:
button has a default font-family. but should use the variable here.